docs/resources/google_compute_instance_group.md

```diff
@@ -15,8 +15,7 @@ A `google_compute_instance_group` resource block declares the tests for a single
 
     describe google_compute_instance_group(project: 'chef-inspec-gcp', zone: 'europe-west2-a', name: 'gcp-inspec-test') do
       it { should exist }
-      its('name') { should eq 'gcp-inspec-test' }
-      its('zone') { should match 'europe-west2-a' }
+      its('size') { should eq 2 }
```

Comment:
I may not be understanding what this resource does, but it is not a plural resource so why would you expect it to have a size of 2?

Is an instance group a group of instances? So in this case you are checking that the number of instances is 2 in that group?

Reply:
The instance group has 2 instances e.g. 
![screen shot 2018-06-21 at 14 32 27](https://user-images.githubusercontent.com/18705468/41722307-025b3102-7560-11e8-9aaa-b8edd2bf143d.png)